### PR TITLE
hotfixing lending re: petabox 25f16688df95a9a0d0b38593edabbbbf2b7e96a2

### PR DESCRIPTION
**Pull request by mekarpeles**

### CODE REVIEW

File: openlibrary/core/lending.py [hunk 1]
```diff
@@ -479,12 +470,11 @@ def new(identifier, resource_type, user_key, book_key=None):
             'type': '/type/loan',
             'user': user_key,
             'book': book_key,
+            'fulfilled': 1,
             'ocaid': identifier,
             'expiry': expiry,
             'uuid': _uuid,
             'loaned_at': loaned_at,
-            'resource_type': resource_type,
-            'resource_id': resource_id,
```
Comment (mekarpeles) [inline openlibrary/core/lending.py]: we should probably put `resource_id` back in as `"bookreader:%s" % identifier`

Comment (mekarpeles) [reply openlibrary/core/lending.py]: actually, tested, this is not necessary. field is being deprecated

**Comment by mekarpeles** [comment]
@bfalling when possible, please provide a retroactive PR. @rchrd2 + @evanminto -- if we plan on reverting on petabox side, please let me know in advance so I can roll back these changes.

**Comment by LeadSongDog** [comment]
(Please ignore above. No connection to 969.

**Pull request merged**